### DISCUSSION TITLE
Not sure where to start...

### DISCUSSION THREAD

**Discussion by wbreiler** [geerlingguy/raspberry-pi-pcie-devices#275] [General]
I've got a CM4, IO board, and 1x to 16x riser on the way. Is there any way i can possibly help? Any certain devices or types of devices that y'all need to further explore?

**Comment by wbreiler** [comment]
> I've got a CM4, IO board, and 1x to 16x riser on the way. Is there any way i can possibly help? Any certain devices or types of devices that y'all need to further explore?

I've got GPUs and a couple NICs I can experiment with, and I'm open to trying almost anything as long as the cost isn't too huge.

**Comment by TheGuyDanish** [comment]
I'd say the best option is just to look at the issues list, if you have any boards that are on it, doing some testing to see if the Pi will boot with it, if it needs drivers, how to get it running, etc is the best thing to do. If you have any cards that aren't listed, testing those and adding them would also be quite beneficial. Anything you could reasonably test for compatibility with is a good call.

**Comment by wbreiler** [comment]
![image](https://user-images.githubusercontent.com/69875107/137511383-ced8217d-f15c-466d-a27c-e233b8b227b1.png)
CM4 Dev Kit on the way! Can't wait to start!
